packages/synapse-react-client/src/components/ChallengeSubmission/AvailableEvaluationQueueList.stories.tsx

```diff
@@ -8,41 +7,37 @@ import {
 } from '../../mocks/entity/mockEvaluationQueue'
 import AvailableEvaluationQueueList from './AvailableEvaluationQueueList'
 
-const meta = {
+const meta: Meta = {
   title: 'Synapse/Challenge/AvailableEvaluationQueueList',
   component: AvailableEvaluationQueueList,
-  render: args => (
-    <Box maxWidth="600px">
-      <AvailableEvaluationQueueList {...args} />
-    </Box>
-  ),
+  decorators: [
+    Story => (
+      <Box maxWidth="600px">
+        <Story />
+      </Box>
+    ),
+  ],
 } satisfies Meta<typeof AvailableEvaluationQueueList>
 export default meta
 type Story = StoryObj<typeof meta>
 
-const onChangeSelectedEvaluation = (evaluation: Evaluation | null) => {
-  return console.log(
-    evaluation ? `Selected ${evaluation.name!}` : 'No evaluation selected',
-  )
-}
-
 export const NoneAvailable: Story = {
   args: {
     evaluations: [],
-    onChangeSelectedEvaluation,
   },
 }
 
+const markdownText = '*markdown* given to the **component**'
 export const OneAvailable: Story = {
   args: {
-    evaluations: [mockEvaluationQueue],
-    onChangeSelectedEvaluation,
+    evaluations: [
+      { ...mockEvaluationQueue, submissionInstructionsMessage: markdownText },
+    ],
```

Comment:
Handle markdown in `submissionInstructionsMessage` -- 

<img width="386" alt="image" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/80dab299-d011-4c50-aac8-6b9c13667367">